src/connectivity.jl

```diff
@@ -536,17 +536,31 @@ end
 """
     isgraphical(degs)
 
-Return true if the degree sequence `degs` is graphical, according to
-[Erdös-Gallai condition](http://mathworld.wolfram.com/GraphicSequence.html).
+Return true if the degree sequence `degs` is graphical
+According to Erdös-Gallai theorem, a degree sequence
+```math
```

Comment:
Does this work? We've been using embedded LaTeX for this in the past..

Reply:
I wanted to ask that,I just guessed it should work,I don't know how to test it.

Reply:
It does work here https://github.com/JuliaGraphs/LightGraphs.jl/blob/1b86c8762861da9259ded620e94d42f7b8e050b8/src/community/modularity.jl#L15

Reply:
I mean it is displayed correctly in the docs

Reply:
That's fantastic. OK, but the part of the docstring under performance is broken (not your fault but it's an opportunity to fix it). Here's what it currently looks like:

![screen shot 2019-02-27 at 10 55 39](https://user-images.githubusercontent.com/941359/53515291-771a1880-3a7e-11e9-81b3-4aeaf472d133.png)

Reply:
Is there a convention to write latex equations,I am a bit confused with mathcal and math.

Reply:
Does LightGraps use https://github.com/JuliaDocs/Documenter.jl ?

Reply:
Here -
![image](https://user-images.githubusercontent.com/22577291/53517395-7f149f80-3af4-11e9-9cd4-1ba1fbad6079.png)
fixed it (works on my machine atleast), ran docs/make.jl to generate the docs.

Reply:
Interestingly, latex works directly without mathcal

Reply:
`\mathcal` is just a latex command for typesetting letters in a certain cursive way. Similar how `\mathbb` draws letters with double lines.

Reply:
Oh okay I have no knowledge about latex